changelogs/unreleased/iangneal__LLZK-302-Replace-Signal-struct-with-column-0-attribute.yaml

```
@@ -0,0 +1,3 @@
+deprecated:
+  - Removed built-in `Signal` component in favor of using column=0 attribute on fields
```

If `column` is specified at all, it's a signal, right?
```suggestion
  - Removed built-in `Signal` component in favor of using `column` attribute on fields. If there is no offset, `column=0` can be used.
```

---

Oh, I think I misunderstood the changes needed for the column attribute; currently `column` is a unit attribute. Are we saying we need to update `column` to be an integer attribute that specifies an offset? I was assuming `column=0` meant `column=false`.